Dockerfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.17.0 as builder
+FROM golang:1.19.0 as builder
```

Comment:
same here as well

Reply:
This one doesn't look to ever be updated by ART, it's the upstream one. We only use `openshift/Dockerfile.rhel`.

Reply:
oops, yeah, i misread that one.